packages/indy-sdk/src/storage/IndySdkStorageService.ts

```diff
@@ -0,0 +1,324 @@
+import type { IndySdkWallet } from '../wallet/IndySdkWallet'
+import type {
+  BaseRecordConstructor,
+  AgentContext,
+  BaseRecord,
+  TagsBase,
+  Query,
+  StorageService,
+} from '@aries-framework/core'
+import type { default as Indy, WalletQuery, WalletRecord, WalletSearchOptions } from 'indy-sdk'
```

Comment:
Same comment as before. (`IndySdk` vs `Indy`)

Reply:
Will update, needs to be Sdk everywhere